plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/panel/GitPanelViewImpl.ui.xml

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
any reason do empty panel with UI binder ?

Reply:
Well, this panel is a place to insert another widget. It is simpler to just use it in the view than overload (even a bit) our code. To me simpler is better. As about UI binder itself, we'll definitely use it for adding more tools in the panel.